gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-all.zip
```

Comment:
why does this reordering happen?

Reply:
This is written by the `gradlew wrapper` task itself.  I just accepted what it wrote.  I never typed this